Changelog.md

```diff
@@ -1,5 +1,9 @@
 # FOSSA CLI Changelog
 
+## Unreleased
+
+- Maven: Fixes a bug where running depgraph would sometimes fail dependending on the local Maven repository's state
```

Comment:
Let's add PR link in the changelog: 


- Maven: Fixes a bug where running depgraph would sometimes fail dependending on the local Maven repository's state ([#988](https://github.com/fossas/fossa-cli/pull/988/files))